lib/app.js

```diff
@@ -0,0 +1,25 @@
+const { createServer } = require('net');
+const ChatRoom = require('./ChatRoom');
+
+const chatRoom = new ChatRoom();
+
+const server = createServer(client => {
```

Comment:
Cannot start server because app is not being exported as a function. Try this syntax:
```suggestion
module.exports = () => {
  const chatRoom = new ChatRoom();
  return createServer(client => { ... });
  };
```